custom_images/redis-3.2/sentinel/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM debian:jessie-slim
+
+ADD ["https://github.com/18f/dig-a/releases/download/v1.0.1/dig-a_linux_amd64", "/dig-a"]
+ADD ["https://github.com/18f/dig-srv/releases/download/v1.0.0/dig-srv_linux_amd64", "/dig-srv"]
```

Comment:
I think we should get rid of using dig-srv and dig-a so we have one fewer dependency, and just use stock `dig`. The dig-srv command is identical to `dig srv +nocmd query +noall +answer`, e.g.:

```sh
./dig-srv _redis-server._tcp.redis-server.default.svc.cluster.local
_redis-server._tcp.redis-server.default.svc.cluster.local. 0 IN SRV 10 33 6379 redis-server-2.redis-server.default.svc.cluster.local.
_redis-server._tcp.redis-server.default.svc.cluster.local. 0 IN SRV 10 33 6379 redis-server-0.redis-server.default.svc.cluster.local.
_redis-server._tcp.redis-server.default.svc.cluster.local. 0 IN SRV 10 33 6379 redis-server-1.redis-server.default.svc.cluster.local.
```

vs:

```sh 
 dig srv +nocmd  _redis-server._tcp.redis-server.default.svc.cluster.local +noall +answer
_redis-server._tcp.redis-server.default.svc.cluster.local. 30 IN SRV 10 33 6379 redis-server-1.redis-server.default.svc.cluster.local.
_redis-server._tcp.redis-server.default.svc.cluster.local. 30 IN SRV 10 33 6379 redis-server-2.redis-server.default.svc.cluster.local.
_redis-server._tcp.redis-server.default.svc.cluster.local. 30 IN SRV 10 33 6379 redis-server-0.redis-server.default.svc.cluster.local.
```

the difference being that `dig-srv` seems to give a difference (possible wrong) answer for 'priority'

I'm guessing likewise for `dig-a`, although I've not dug into it as much yet.